src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/ExceptionsConfiguration.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.WebApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Exceptions
+{
+    public class ExceptionsConfiguration
+    {
+        /// <summary>
+        /// The list of exception configurations that determine which exceptions should be shown.
+        /// Each configuration is a logical OR, so if any of the configurations match, the exception is shown.
+        /// </summary>
+        [JsonPropertyName("include")]
+        public List<ExceptionFilter> Include { get; set; } = new();
+
+        /// <summary>
+        /// The list of exception configurations that determine which exceptions should be shown.
+        /// Each configuration is a logical OR, so if any of the configurations match, the exception isn't shown.
+        /// </summary>
+        [JsonPropertyName("exclude")]
+        public List<ExceptionFilter> Exclude { get; set; } = new();
+
+        private static bool CheckConfiguration(IExceptionInstance exception, List<ExceptionFilter> filterList, Func<ExceptionFilter, CallStackFrame, bool> evaluateFilterList)
```

Comment:
At best, these should be extension methods in a separate class. Let's not mix serialization classes with business logic.